sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_models.hpp

```diff
@@ -970,4 +970,29 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     IssuerProperties Properties;
   };
 
+  /**
+   * @brief The contact information for the vault certificates.
+   *
+   */
+  struct CertificateContact
+  {
+    /**
+     * @brief Contact email address of.
+     *
+     * @remark Email is required, has to be unique
+     */
+    std::string EmailAddress;
+
+    /**
+     * @brief Contact name.
+     *
+     */
+    Azure::Nullable<std::string> Name;
+
+    /**
+     * @brief Contact phone number.
+     *
+     */
+    Azure::Nullable<std::string> Phone;
```

Comment:
Should this be named `PhoneNumber`?

Reply:
both .Net and Swagger call it Phone 
        /// <summary>
        /// Gets or sets the phone number of the contact.
        /// </summary>
        public string Phone { get; set; }